charts/images.yaml

```diff
@@ -251,6 +251,10 @@ images:
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/telegraf-iptables
   tag: "v0.41.0"
+- name: event-logger
+  sourceRepository: github.com/gardener/logging
+  repository: eu.gcr.io/gardener-project/gardener/event-logger
+  tag: "v0.41.0"
```

Comment:
I restarted the event-logger Pod 3 times and all events in the loki are now duplicated 3 times. IIUC on startup the event-logger prints all events. Isn't this a concern? An event log in loki can be duplicated n times (where n is the number of restarts of the event-logger Pod).

Reply:
I know, but I don't see how can't event-logger remember which events have been logged if it is not a StatefulSet or DaemonSet. Do I have to dedicate a PV only for one offset file?
In the next event-logger release, I will make the event-logger print only events younger than 5 seconds.

Reply:
> Do I have to dedicate a PV only for one offset file?

I don't think we need a dedicated PV only for this.

> In the next event-logger release, I will make the event-logger print only events younger than 5 seconds.

Sounds reasonable. Can you create issue on gardener/logging side about it to make sure we don't forget it?

Reply:
no, it is already added in 0.42.0.